README.md

```diff
@@ -82,6 +82,17 @@ Usage Changes
 Versions 0.2.6 and prior supported both N5 and Zarr output using the `--file_type` option.
 This option is not present in 0.3.0 and later, as only Zarr output is supported.
 
+Versions 0.2.6 and prior used the input file's dimension order to determine the output
```

Comment:
Might be an idea to clarify the difference in how we write down array index order since this will be the first instance that many people will be exposed to this setting. We're taking a "Bio-Formats" style dimension order on the command line, correct?